src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -29,15 +29,11 @@
 #include "squid.h"
 #include "helper/protocol_defines.h"
 
-#include <cstdlib>
-#include <cstring>
+#include <chrono>
 #include <ctime>
-#include <sys/types.h>
-#include <sys/stat.h>
-#include <fcntl.h>
-#if HAVE_UNISTD_H
-#include <unistd.h>
-#endif
+#include <fstream>
+#include <iostream>
```

Comment:
`iostream` is provided by `helper/protocol_defines.h` which are already using `strd::cout` for the provided operations.
```suggestion
```